### DISCUSSION TITLE
Podman scopes

### DISCUSSION THREAD

**Discussion by bugi353** [containers/podman#12448] [Q&A] [answered]
Hi, I am learning podman and using it with oracle linux and gitlab-runner stuff. As i know gitlab-runner run scripts as user gitlabrunner in system. I also prepared scripts for deploying my app. When i deploy app from system level as gitlabrunner using sudo -u gitlabrunner script_name and the run sudo -u gitlabrunner podman ps or other commands I can communicate with containers but when i try to do this  from gitlab scripts I don't see this in user scope. Also i can't communicate with container via podman exec .... I want to add that I run podman containers in rootless mode as root inside container. I found a solution to most of the problems I encountered, but I can't find solution or cause of this.

**Comment by rhatdan** [comment, marked as answer]
Could this be a linger issue? @giuseppe @mheon WDYT?

**Reply by mheon** [reply]
Probably - the use of `sudo -u` exclusively makes me agree. I assume that the user in question did not have `loginctl enable-linger` run?

Any chance you can provide the information from the bug reporting template (`podman info`, `podman version`, what OS you are using, etc)?

**Reply by bugi353** [reply]
Unfortunately I currently don't have access to this machine but i will check it on my other oracle linux server and I will reproduce it but also will check and dive into this loginctl linger issue. Thank you for response.

**Reply by bugi353** [reply]
I checked it on my another oracle linux server and enable-linger for gitlab-runner solved problem. Now i can see all containers from both levels and succesfully run podman exec for containers by gitlab pipline. Than you very much.
